packages/babel-plugin-jest-hoist/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@babel/template": "^7.3.3",
     "@babel/types": "^7.3.3",
-    "@types/babel__core": "^7.0.0",
+    "@types/babel__core": "^7.1.11",
```

Comment:
we don't need to bump this as the type for `pre` and `post` is not included in the generated TS definition

```ts
import { Identifier } from '@babel/types';
import type { PluginObj } from '@babel/core';
declare const _default: () => PluginObj<{
    declareJestObjGetterIdentifier: () => Identifier;
    jestObjGetterIdentifier?: Identifier;
}>;
export default _default;
```